variables.tf

```
@@ -23,3 +23,84 @@ variable "tags" {
   type    = map(string)
   default = null
 }
+
+variable "enable_worm_policy" {
+  type        = bool
+  description = "Enable WORM (Write Once Read Many) policy for the KMS key. Default: false"
+  default     = false
+}
+
+variable "worm_policy_years" {
+  type        = number
+  description = "Number of years for the WORM policy retention period. Only used if enable_worm_policy is true. Default: 1"
+  default     = 1
```

In context of https://github.com/iits-consulting/terraform-opentelekomcloud-obs-restricted/pull/4/files#r2405039072


```suggestion
  description = "[Optional] Number of years for the WORM policy retention period. Only used if enable_worm_policy is true."
```